### DISCUSSION TITLE
PiZero1: RTLSDR not enabled

### DISCUSSION THREAD

**Discussion by buentAir** [rtl-airband/RTLSDR-Airband#319] [Q&A] [answered]
RTLSDR cannot be enabled on pi zero, but had it active some time ago. Am i missing something?

`-   - librtlsdr:               requested: TRUE, enabled: FALSE`

Saw the error:
`-- Checking for module 'librtlsdr' --   No package 'librtlsdr' found`

How I generated:
`cmake -DPLATFORM=rpiv1 -DNFM=ON -DRTLSDR=TRUE -DMIRISDR=FALSE -DSOAPYSDR=FALSE -DPULSEAUDIO=FALSE ../`

The dongle was installed as on the Pi 3:
`sudo apt-get install rtl-sdr`

**Comment by starvald** [comment, marked as answer]
try installing librtlsdr-dev

**Comment by buentAir** [comment]
Great, that's how it works. Thanks very much!
Only now see that this is mentioned in the dependencies, shameful.
